### DISCUSSION TITLE
Compatibility with older setuptools

### DISCUSSION THREAD

**Discussion by yan12125** [aio-libs/aiobotocore#1193] [General]
aiobotocore now requires setuptools>=72.2.0 to build: https://github.com/aio-libs/aiobotocore/commit/0e6dd625d248f11bbb22b6208c21e3fd39d0e066. However, on Arch Linux [the python-setuptools package](https://archlinux.org/packages/extra/any/python-setuptools/) is stuck at an old version due to its complexity. I tried removing the version constraint `>=72.2.0`, and the built package seems fine. Is it really necessary? Thanks.

**Comment by jakob-keller** [comment]
Thank you for reporting. The version constraint is not arbitrary, but could potentially be relaxed. What version of setuptools would be available in your context? I could then do some testing.

**Reply by yan12125** [reply]
Thank you for considering lowering the bound. As of writing, Arch Linux comes with setuptools 69.5.1

**Reply by jakob-keller** [reply]
I checked with setuptools 69.5.1 and the resulting distribution packages are nearly identical and fully functional. I will perform additional tests and then try to establish a less strict, yet safe constraint.

**Reply by jakob-keller** [reply]
#1194 is addressing the issue. Do you need a new release on PyPI or is this sufficient?

**Reply by yan12125** [reply]
Thank you! I don't need a new release on PyPI. I can cherry-pick the change in python-aiobotocore package for Arch Linux .
